Source/HelixToolkit.Wpf.SharpDX/Utilities/ObjReader.cs

```diff
@@ -1180,13 +1180,13 @@ private Material CreateMaterial(string texturePath)
                     //AmbientMap = this.AmbientMap,
 
                     DiffuseColor = this.Diffuse,
-                    DiffuseMap = (this.DiffuseMap == null) ? null : LoadImage(this.DiffuseMap),
+                    DiffuseMap = (this.DiffuseMap == null) ? null : LoadImage(Path.Combine(texturePath, this.DiffuseMap)),

```

Comment:
I think it would be better to set the path in line 925, and remove the texturePath from this method.